deploy/crds/kieapp.crd.yaml

```diff
@@ -497,8 +497,8 @@ spec:
                               be mounted.
                             type: string
                           sshSecret:
-                            description: Secret to use for ssh key and known hosts
-                              file.
+                            description: 'Secret to use for ssh key and known hosts
+                              file. The secret must contain two files: id_rsa and known_hosts'
```

Comment:
make that change here instead and then run `make test` to generate the crd.
https://github.com/kiegroup/kie-cloud-operator/blob/8a00102178cf17c2f23266e994638eab485947d8/pkg/apis/app/v2/kieapp_types.go#L303